frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -33,71 +36,64 @@ export function MultiAutocomplete({
     finalValue: "",
     onChange: onSearchChange,
   });
-  const [lastSelectedValues, setLastSelectedValues] = useState(selectedValues);
-  const [isFocused, setIsFocused] = useState(false);
-  const visibleValues = isFocused ? lastSelectedValues : selectedValues;
```

Comment:
I found no discernible difference between my implementation and this one that uses `lastSelectedValues` and `selectedValues`.

It does make me wonder: am I missing something here?